Dockerfile

```diff
@@ -6,13 +6,25 @@ VOLUME /output
 RUN mkdir -p /jenkins_node_scanner
 WORKDIR /jenkins_node_scanner
 
+RUN apt-get update \
+    && apt-get install -y supervisor=3.3.5-1 wget=1.20.1-1.1 --no-install-recommends \
+    && wget -q -O - https://github.com/papertrail/remote_syslog2/releases/download/v0.17/remote_syslog_linux_amd64.tar.gz \
```

Comment:
I don't think that the `wget` and the `apt-get` stuff should be on the same stripe. Also, it makes the diff look weird a few commits later on. Please move this to a separate `RUN` command.